src/main/java/core/basesyntax/model/Report.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.model;
+
+public class Report {
+    private static final int DEFAULT_AMOUNT = 0;
+    private int supply;
+    private int buy;
+
+    public Report() {
+        this(DEFAULT_AMOUNT, DEFAULT_AMOUNT);
+    }
+
+    public Report(int totalSupply, int totalBuy) {
+        this.supply = totalSupply;
+        this.buy = totalBuy;
+    }
+
+    public void incrementSupply(int increment) {
+        this.supply += increment;
+    }
+
+    public void incrementBuy(int increment) {
+        this.buy += increment;
+    }
+
+    public int getSupply() {
+        return supply;
+    }
+
+    public int getBuy() {
+        return buy;
+    }
+
+    public int getResult() {
+        return supply - buy;
+    }
```

Comment:
The `Report` class seems well-structured and follows Java naming conventions. However, the class itself could be made more flexible by allowing the report fields to be set not only through the constructor but also with setter methods, if the use case of the application requires such behavior. This is not a requirement but something to consider depending on how the `Report` class will be used in the larger application.